config/v1/types_ingress.go

```diff
@@ -26,6 +26,8 @@ type IngressSpec struct {
 	// domain is used to generate a default host name for a route when the
 	// route's host name is empty.  The generated host name will follow this
 	// pattern: "<route-name>.<route-namespace>.<domain>".
+	//
+	// Once set, domain cannot be updated.
```

Comment:
@smarterclayton, this is true of `ingresscontroller.spec.domain` — is it also true here? ingress-operator won't actively observe the change, but the value will be used to drive the default domain for any ingresscontroller you create that lacks a `.spec.domain` value, so in that way it's honored, but it seems misleading to claim we support updating it in a deliberate/reasoned way at this point.

Reply:
Specify exactly what happens and what is supported.

Reply:
I added an additional point about another way in which the value is used, and clarified that changing the value is not supported.